### ISSUE TITLE
Problemă de extensie

### ISSUE THREAD

**Issue by webofiice** [simoebenhida/laravel-intellisense#29]
- Issue Type: `Bug`
- Extension Name: `laravel-intellisense`
- Extension Version: `0.1.9`
- OS Version: `Windows_NT x64 10.0.18363`
- VSCode version: `1.52.1`

:warning:  :warning:
```json 
{
	"messages": [],
	"activationTimes": {
		"codeLoadingTime": 44,
		"activateCallTime": 29,
		"activateResolvedTime": 862,
		"activationReason": {
			"startup": true,
			"extensionId": {
				"value": "mohamedbenhida.laravel-intellisense",
				"_lower": "mohamedbenhida.laravel-intellisense"
			},
			"activationEvent": "*"
		}
	},
	"runtimeErrors": [
		{
			"name": "SyntaxError",
			"message": "Unexpected end of JSON input"
		}
	]
}
```

**Comment by simoebenhida** [comment]
It should be fixed on 0.2.0 version

**Comment by SeyidTakele** [comment]
I still have this issue
